code/mob/living/critter/flockcritter_parent.dm

```diff
@@ -142,33 +142,30 @@
 
 /mob/living/critter/flock/understands_language(var/langname)
 	if (langname == say_language || langname == "feather" || langname == "english")
-		return 1
-	return 0
+		return TRUE
+	return FALSE
 
-// common features all flock bots get include automatic fire extinguishers
```

Comment:
useful comment

Reply:
This one is probably ok

Reply:
I removed it as there's already a comment in Life() for exinquishers